### ISSUE TITLE
Allowing only Ruby >= 1.9.1 versions

### ISSUE THREAD

**Issue by Kulgar** [lang/unicode_utils#2]
Hi,

I wanted to install this gem on a ree (Ruby Entreprise Edition) version of Ruby. But as it requires ruby >= 1.9.1, it can't be installed. 

Could you do a quick fix to allow the installation on a ree version of Ruby ? 

Thx !

Kulgar.

**Comment by lang** [comment]
AFAIK REE is only compatible with Ruby 1.8. UnicodeUtils depends on Ruby 1.9 features, most important efficient iteration over codepoints (String#each_codepoint).

Unless there's an efficient implementation/replacement of String#each_codepoint, there will be no UnicodeUtils support.

**Comment by Kulgar** [comment]
Yes I'm sorry, I discovered yesterday that Ree was going to die because Ruby 1.9.1+ rocks. I forgot to close the issue here...

I've lost some time before discovering that Ree is reaching end of life... Their programmers don't put this important information on the ree's page... I'm sorry if I troubled you with this issue that - in fact - is really not one... Trying to use Ree instead of Ruby 1.9.3 was the real issue. ^^'

Thanks anyway! :)
